guides/06_Meeting_Events.md

```diff
@@ -128,6 +133,7 @@ The following table describes attributes for a meeting.
 |`retryCount`|The number of connection retries performed during the meeting.<br><br>Unit: Count|`meetingStartSucceeded`, `meetingStartFailed`, `meetingEnded`, `meetingFailed`
 |`signalingOpenDurationMs`|The time taken for opening a WebSocket connection.<br><br>Unit: Milliseconds|`meetingStartSucceeded`, `meetingStartFailed`, `meetingEnded`, `meetingFailed`
 |`iceGatheringDurationMs`|The time taken for connection's ICE gathering state to complete.<br><br>Unit: Milliseconds|`meetingStartSucceeded`, `meetingStartFailed`, `meetingEnded`, `meetingFailed`
+|`selfAttendeePresentDurationMs`|The time taken for attendees' first audio packet to be received by the Chime Services.<br><br>Unit: Milliseconds|`selfAttendeePresent`
```

Comment:
- Since the description is also an implementation detail and not accurate, how about the following?
  > The time taken for the attendee to be present in the meeting.
- I see that you include this attribute in the following events. List them in the last column.
  - `meetingEnded`
  - `meetingFailed`
  - `meetingStartFailed` (<- Do not add. See my comment in `DefaultAudioVideoController`.)